index.html

```diff
@@ -133,15 +133,15 @@ <h4>Suzaan clarke</h4>
   <div class="container">
       <div class="title"><h2>Contact Us</h2></div>
       <form>
-          <input type="text" id="name" class="field" placeholder="Your Name">
-          <input type="text" id="email" class="field" placeholder="Your Email">
-          <textarea id="message" placeholder="Your Message" class="field"></textarea>
+          <input type="text" class="field" placeholder="Your Name">
```

Comment:
make the class name more descriptive, just as before: name and email, and if you want add same style for them grouping them in css

Reply:
Done